tests/cmd/e2e/main.go

```diff
@@ -32,14 +32,10 @@ func main() {
 	logs.InitLogs()
 	defer logs.FlushLogs()
 
-	conf := tests.NewConfig()
-	err := conf.Parse()
-	if err != nil {
-		glog.Fatalf("failed to parse config: %v", err)
-	}
+	conf := tests.ParseConfigOrDie()
+	conf.ChartDir = "/charts"
```

Comment:
The default value should be set before configuration file load, otherwise this will override anything

Reply:
it is right, e2e test must set `conf.ChartDir` as "/charts" becuase the charts file is add by CI.